### PR TITLE
[Holy Paladin] Stat weight scaling

### PR DESCRIPTION
**Pull request by abelito75**
Just as the title reads

### CODE REVIEW

File: src/parser/paladin/holy/CHANGELOG.js [hunk 1]
```diff
@@ -7,8 +7,9 @@ import { change, date } from 'common/changelog';
 
 // prettier-ignore
 export default [
-  change(date(2020, 8, 27), <> Updated core Holy Paladin for prepatch. </>, Abelito75),
-  change(date(2020, 8, 27), <> Enabled a statistic for the minor of <SpellLink id={SPELLS.LUCID_DREAMS.id} />. </>, Putro),
+  change(date(2020, 8, 27), <>Updated stat weight scaling. </>, Abelito75),
```
Comment (MartijnHols) [inline src/parser/paladin/holy/CHANGELOG.js:10]: old

Comment (abelito75) [reply src/parser/paladin/holy/CHANGELOG.js:10]: oh you right. let me fix that

File: src/parser/paladin/holy/modules/StatValuesSpellInfo.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ export default {
   [SPELLS.LIGHT_OF_DAWN_HEAL.id]: {
     int: true,
     crit: true,
-    hasteHpct: true,
+    hasteHpct: false,
```
Comment (MartijnHols) [inline src/parser/paladin/holy/modules/StatValuesSpellInfo.js:28]: Aren't you able to cast more HP generators if you have more Haste and thus cast more LoD?

Comment (MartijnHols) [reply src/parser/paladin/holy/modules/StatValuesSpellInfo.js:28]: Some generators you cast for the generation (e.g. HL on beacons), others you get for spells you'd have cast regardless. Those have different value too.

File: src/parser/paladin/holy/modules/StatValuesSpellInfo.js [hunk 3]
```diff
@@ -140,4 +140,18 @@ export default {
     mastery: true,
     vers: true,
   },
+  [SPELLS.WORD_OF_GLORY.id]: {
+    int: true,
+    crit: true,
+    mastery: true,
```
Comment (MartijnHols) [inline src/parser/paladin/holy/modules/StatValuesSpellInfo.js:146]: haste??

**Pull request merged**